Makefile

```diff
@@ -233,6 +233,28 @@ deploy-test: _verify_minikube generate-keys
 	export ENABLE_AUTH="True" && export DUMMY_IGNITION="True" && \
 	$(MAKE) _update-minikube deploy-wiremock deploy-all
 
+generate-onprem-environment:
+	echo "POSTGRESQL_DATABASE=installer" > onprem-environment
+	echo "POSTGRESQL_PASSWORD=admin" >> onprem-environment
+	echo "POSTGRESQL_USER=admin" >> onprem-environment
+	echo "DB_HOST=127.0.0.1" >> onprem-environment
+	echo "DB_PORT=5432" >> onprem-environment
+	echo "DB_USER=admin" >> onprem-environment
+	echo "DB_PASS=admin" >> onprem-environment
+	echo "DB_NAME=installer" >> onprem-environment
+	echo "SERVICE_BASE_URL=http://127.0.0.1:8090" >> onprem-environment
+	echo "DEPLOY_TARGET=onprem" >> onprem-environment
+	echo "DUMMY_IGNITION=false" >> onprem-environment
+	echo "OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> onprem-environment
```

Comment:
Please add PUBLIC_CONTAINER_REGISTRIES to whitelist registries that don't require authentication.

Reply:
Done!